.github/workflows/Build.yaml

```diff
@@ -138,13 +138,16 @@ jobs:
         run: ./gradlew packageDemoDebug packageDemoDebugAndroidTest
 
       - name: Run instrumented tests with GMD
-        run: ./gradlew cleanManagedDevices --unused-only &&
-          ./gradlew ciDemoDebugAndroidTest -Dorg.gradle.workers.max=1
+        run: ./gradlew ciDemoDebugAndroidTest --no-parallel --max-workers=1
```

Comment:
`--no-parallel --max-workers=1`

This looks like it might substantially increase the execution time.

Reply:
Since we are not sharing in any way, I don't think this will have any impact.